containers/Ideas/Phase.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {View} from 'react-native';
+
+import {TextSourceSans} from '../../components/TextSourceSans';
+import { styles } from './Phase.styles';
+
+export const Phase = props => {
+  const {activePhase, futurePhases, pastPhases} = props;
+  const startDate = (futurePhases) ? futurePhases[0].start_date : null;
+  const endDate = (pastPhases) ? pastPhases[pastPhases.length-1].end_date : null;
+
+  return (
+    activePhase ?  (
```

Comment:
Yeah, all the `?` and `:` is hard to understand, but looking at it for a while it makes sense. :woman_shrugging: I am fine with leaving it like that.